### PR TITLE
enable snyk code by default

### PR DESCRIPTION
**Pull request by yechielw**
the language server is quite unless without snyk code

### CODE REVIEW

File: lua/lspconfig/server_configurations/snyk_ls.lua [hunk 1]
```diff
@@ -6,7 +6,9 @@ return {
     root_dir = util.root_pattern '.git',
     single_file_support = true,
     settings = {},
-    init_options = {},
+    init_options = {
+      activateSnykCode = "true",
```
Comment (glepnir) [inline lua/lspconfig/server_configurations/snyk_ls.lua:10]: this boolean as string type to send ?

Comment (yechielw) [reply lua/lspconfig/server_configurations/snyk_ls.lua:10]: this is how it is used in snyk's documentation,
image

https://docs.snyk.io/integrate-with-snyk/use-snyk-in-your-ide/snyk-language-server

Comment (glepnir) [reply lua/lspconfig/server_configurations/snyk_ls.lua:10]: thanks just fix lint

**Comment by yechielw** [comment]
Thanks

**Pull request merged**